src/library_dylink.js

```diff
@@ -210,6 +202,14 @@ var LibraryDylink = {
     loadedLibNames: {},
   },
 
+  $dlSetError: ['$stringToNewUTF8', '___dl_seterr'],
```

Comment:
can the dep be of `intArrayFromString` directly?

Reply:
Oh yes, that was from an earlier version.